server/src/main/java/io/deephaven/server/console/ConsoleServiceGrpcImpl.java

```diff
@@ -81,6 +81,8 @@ public ConsoleServiceGrpcImpl(final TicketRouter ticketRouter,
     public void getConsoleTypes(final GetConsoleTypesRequest request,
             final StreamObserver<GetConsoleTypesResponse> responseObserver) {
         GrpcUtil.rpcWrapper(log, responseObserver, () -> {
+            // for legacy reasons, this method is used before the session is created
```

Comment:
I think in the brave-new exec context world, we can un-do this pattern and allow multiple sessions or language selection from the GUI.